### PR TITLE
Fix "14dB" RF amplification in FAQ

### PR DESCRIPTION
**Pull request by Sasszem**
Also added a comment on why is this value often quoted wrong

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -116,23 +116,24 @@ HackRF (both Jawbreaker and One) provides three different analog gain controls o
...
 - IF ("lna", 0 to 40 dB in 8 dB steps)
 - baseband ("vga", 0 to 62 dB in 2 dB steps)
  
 The two TX gain controls are at these stages:
 
-- RF (0 or 14 dB)
+- RF (0 or ~11 dB)
 - IF (0 to 47 dB in 1 dB steps)
 
+Note: in some documents, the RF gain was erroneously quoted to be 14dB. The confusion was based on the fact that the MGA-81563 amplifier is advertised as a "14 dBm" amplifier, but that specifies its output power, not its amplification. See `Martin Ling's comment on issue #1059<https://github.com/greatscottgadgets/hackrf/issues/1059#issuecomment-1060038293>`__ for some details!
```
Comment (mossmann) [inline docs/source/faq.rst:128]: Can you please add a space before "dB"?

**Review by mossmann — CHANGES_REQUESTED** [review]
This looks great, but there is a tiny change I would like. Is there a reason this is a draft?

File: docs/source/faq.rst [hunk 2]
```diff
@@ -116,23 +116,24 @@ HackRF (both Jawbreaker and One) provides three different analog gain controls o
...
 - IF ("lna", 0 to 40 dB in 8 dB steps)
 - baseband ("vga", 0 to 62 dB in 2 dB steps)
  
 The two TX gain controls are at these stages:
 
-- RF (0 or 14 dB)
+- RF (0 or ~11 dB)
 - IF (0 to 47 dB in 1 dB steps)
 
+Note: in some documents, the RF gain was erroneously quoted to be 14 dB. The confusion was based on the fact that the MGA-81563 amplifier is advertised as a "14 dBm" amplifier, but that specifies its output power, not its amplification. See `Martin Ling's comment on issue #1059<https://github.com/greatscottgadgets/hackrf/issues/1059#issuecomment-1060038293>`__ for some details!
```
Comment (mossmann) [inline docs/source/faq.rst:128]: A space is needed before "<https" for the link to work.

**Review by mossmann — CHANGES_REQUESTED** [review]
Oops. I missed something in my earlier review:

**Comment by Sasszem** [comment]
Fixed it 
I see CI is now failing, but I'm not sure it's related to my changes.

**Comment by mossmann** [comment]
> I see CI is now failing, but I'm not sure it's related to my changes.

Yeah, we have an unreliable script in our hardware CI.

**Review by mossmann — APPROVED** [review]
Perfect. Thank you for the contribution!

**Pull request merged**